src/app/cli/src/output/output_config.rs

```diff
@@ -98,6 +102,25 @@ pub enum OutputFormat {
     /// A pretty human-readable table
     #[clap(name = "table")]
     Table,
+
+    /// Parquet columnar storage. Only available when exporting to file(s)
+    #[clap(name = "parquet")]
+    Parquet,
+}
+
+impl Display for OutputFormat {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        let str_val = match *self {
+            OutputFormat::Csv => "csv",
+            OutputFormat::Json => "json",
+            OutputFormat::NdJson => "ndjson",
+            OutputFormat::JsonSoA => "json-soa",
+            OutputFormat::JsonAoA => "json-aoa",
+            OutputFormat::Table => "table",
+            OutputFormat::Parquet => "parquet",
```

Comment:
And why custom display was necessary? Can't we live with the automatic version?

Reply:
`strum`?

Reply:
We use `fmt` when printing error message, where we need to display supported formats. And we need custom formatter, to support all custom mappings like "NdJson" -> "ndjson", "JsonSoA" -> "json-soa"